.gitignore

```diff
@@ -2,6 +2,7 @@
 .DS_Store
 **/worker/generated/*
 worker-sandbox/build
+examples/**/build
```

Comment:
I added this so that no build folders get checked in, though a single `**/build` would also work. I feel like that would be a bit less clear though.